lib/sfnt.js

```diff
@@ -15,7 +15,10 @@ function Font() {
   this.italicAngle = 0;
   this.familyClass = 0; // No Classification
   this.familyName = '';
-  this.fsSelection = 0x40; // Characters are in the standard weight/style for the font.
+  // 0x40 - REGULAR - Characters are in the standard weight/style for the font
+  // 0x80 - USE_TYPO_METRICS - use OS/2.sTypoAscender - OS/2.sTypoDescender + OS/2.sTypoLineGap as the default line spacing
+  // https://docs.microsoft.com/en-us/typography/opentype/spec/os2#fsselection
+  this.fsSelection = 0x40 | 0x80;
   // Non zero value can cause issues in IE, https://github.com/fontello/svg2ttf/issues/45
```

Comment:
If Bit 7 is enabled, the version seems to be changed to 4
https://docs.microsoft.com/en-us/typography/opentype/spec/os2#fsselection
>Bit 7 was defined in version 4. For new fonts, vendors are encouraged to use a version 4 or later OS/2 table and to have bit 7 set.

https://github.com/fontello/svg2ttf/blob/f1b0dbbe6bfbaed83fd8c7be1cda1ebf7404e593/lib/ttf/tables/os2.js#L27